robot_calibration/include/robot_calibration/ceres/camera3d_to_arm_error.h

```diff
@@ -106,11 +106,21 @@ struct Camera3dToArmError
                                      CalibrationOffsetParser* offsets,
                                      robot_calibration_msgs::CalibrationData& data)
   {
+    int index = -1;
+      for (size_t k =0; k < data.observations.size() ; k++)
+      {
+        if ( data.observations[k].sensor_name == "camera")
+        {
+          index = k;
+          break;
+        }
+      }
+  
```

Comment:
need to add a check -- if index is -1 then we have to exit with error (rather than segfault)